recipes/java-jdk/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "8.45.14" %}
+
+package:
+  name: java-jdk
+  version: {{ version }}
+
+# Cannot use source section because of cookies blocking download
+#source:
+#  fn: jre-8u25-linux-x64.tar.gz
+#  url: http://download.oracle.com/otn-pub/java/jdk/8u25-b17/jre-8u25-linux-x64.tar.gz
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - curl  # [win]
+
+test:
+  commands:
+    - java -help
+
+about:
+  home: http://www.java.com/en/
+  license: "Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX"
```

Comment:
Azul has one that is GPL: https://www.azul.com/products/zulu/

Continuum is currently repackaging that for java jdk.